### ISSUE TITLE
text-decoration not inherited from tspan

### ISSUE THREAD

**Issue by LaurenzV** [RazrFalcon/resvg#674]
Consider the following SVG:
```svg
<svg width="200" height="200" viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
    <defs/>
    <text id="text1" font-family="Noto Sans" font-size="48" x="50" y="100">
        <tspan text-decoration="overline" fill="#000000" stroke="#FF0000">
            <tspan>Text</tspan>
        </tspan>
    </text>
</svg>
```

When rendering with resvg, the overline will not appear at all. If I put it into the `text` element instead, it will appear. Shouldn't it also be inherited when it appears in a t-span element? Especially since the fill and stroke do seem to be inherited. At least in Chrome and Safari it seems to show up that way. Is this a bug or am I missing something about how inheritance works in this case?

**Comment by RazrFalcon** [comment]
#410 ?

**Comment by RazrFalcon** [comment]
`text-decoration` is insanely hard to parse and apply: https://razrfalcon.github.io/notes-on-svg-parsing/text/text-decoration.html
